selene-vscode/src/extension.ts

```diff
@@ -160,9 +160,28 @@ export async function activate(context: vscode.ExtensionContext) {
         diagnosticsCollection.set(document.uri, diagnostics)
     }
 
-    vscode.workspace.onDidSaveTextDocument(lint)
+    let disposable: vscode.Disposable
+    function listenToChange() {
+        if (disposable) {
+            disposable.dispose()
```

Comment:
disposable?.dispose()

Reply:
On second thought, just dispose it when the configuration is changed. There's no real reason to handle it here.